test/package.mjs

```diff
@@ -236,7 +236,7 @@ test('Every rule has a doc with the appropriate content', t => {
 		// Decide which notices should be shown at the top of the doc.
 		const expectedNotices = [];
 		const unexpectedNotices = [];
-		if (index.configs.recommended.rules[`unicorn/${ruleName}`]) {
+		if (index.configs.recommended.rules[`unicorn/${ruleName}`] === 'error') {
 			expectedNotices.push('configRecommended');
```

Comment:
`!= 'off'`，, I believe `template-indent` was configured as `warn`.

Reply:
Thanks, fixed

Reply:
Had to switch to `includes` to avoid [no-negated-condition](https://eslint.org/docs/rules/no-negated-condition).